app/config/base.json

```diff
@@ -4,7 +4,7 @@
 	"useLocalSessions": true,
 	"welcomePage": {
 		"avatarSelectionEnabled": true,
-		"backgroundColor": "black",
+		"backgroundColor": "#121212",
```

Comment:
So in a next step, would this backgroundColor plus additional styling be specified in an scss file that overrides `_default.scss`?

Reply:
yeah, something along those lines

Reply:
Should we remove this from the default json config since it's now specified in the default scss?